Mage/Module.cs

```diff
@@ -67,28 +83,46 @@ public void SetupStaticData(Action<Exception> cb)
 		}
 
 
-		//
-		protected virtual string CommandPrefix
-		{
-			get { return null; }
-		}
+		// The module name as defined on the remote MAGE server
+		protected abstract string CommandPrefix { get; }
```

Comment:
again, the same as below you're enforcing this to be defined. IIRC the current implementation allows this to remain optional to avoid bloat in modules that have no usercommands.
CC: @nullorvoid